js/ui/calendar/ui.calendar.js

```diff
@@ -706,16 +779,14 @@ const Calendar = Editor.inherit({
             return;
         }
 
-        const contouredDate = this._view.option('contouredDate');
-
         this.option('zoomLevel', nextView);
-        this.option('currentDate', contouredDate || this._view.option('date'));
 
         this._renderNavigator();
 
-        this._animateShowView().done((function() {
-            this._setViewContoured(contouredDate);
-        }).bind(this));
+        this._animateShowView();
```

Comment:
Won't there be any problems with code execution until the animation is complete?

Reply:
There is only focused date contouring after animation start. If we wait until animation stops working, contour will appear after animation and this looks weird :)